code/game/machinery/recycler.dm

```diff
@@ -19,7 +19,7 @@
 	var/item_recycle_sound = 'sound/items/welder.ogg'
 
 /obj/machinery/recycler/Initialize()
-	AddComponent(/datum/component/material_container, list(MAT_METAL, MAT_GLASS, MAT_PLASMA, MAT_SILVER, MAT_GOLD, MAT_DIAMOND, MAT_URANIUM, MAT_BANANIUM, MAT_TITANIUM))
+	AddComponent(/datum/component/material_container, list(MAT_METAL, MAT_GLASS, MAT_PLASMA, MAT_SILVER, MAT_GOLD, MAT_DIAMOND, MAT_URANIUM, MAT_BANANIUM, MAT_TITANIUM, MAT_BLUESPACE, MAT_PLASTIC))

```

Comment:
There should probably be a way to disable the `attackby` portion of the material container component; there may be several instance where you want all the benefits of the material container, but you don't want anyone to be able to insert things into the object that possess the component.

Reply:
flags_2 & NO_MAT_REDEMPTION_2

Reply:
No it's not; that's something you put on an *object* you place into the material container; if the object has that, then it won't accept it.

What I'm saying is a universal way to block insertion of ALL objects onto the component; that'd require a var on the material component itself.